blueocean-dashboard/src/main/js/components/testing/TestCaseResultRow.jsx

```diff
@@ -0,0 +1,98 @@
+import React, { Component, PropTypes } from 'react';
+import { action, observable } from 'mobx';
+import { observer } from 'mobx-react';
+import { Fetch } from '@jenkins-cd/blueocean-core-js';
+import { ResultItem, StatusIndicator } from '@jenkins-cd/design-language';
+import moment from 'moment';
+import TestDetails from './TestDetails';
+
+/* eslint-disable max-len */
+
+@observer
+export default class TestCaseResultRow extends Component {
+
+    propTypes = {
+        testCase: PropTypes.object,
+        translation: PropTypes.func,
+        locale: PropTypes.string,
+    };
+
+    componentWillMount() {
+        this.setState({ isFocused: false, attemptedFetchForStdOutStdErr: false });
+    }
+
+    @action
+    setStdout(stdout) {
+        this.stdout = stdout;
+    }
+
+    @action
+    setStderr(stderr) {
+        this.stderr = stderr;
+    }
+
+    @observable stdout = null;
+    @observable stderr = null;
+
+    render() {
+        const { testCase: t, translation, locale = 'en' } = this.props;
```

Comment:
Shortening `testCase` to `t` kinda limits readability here, and isn't a great idea since in many other components we've shortened the `translation` function to `t`.